src/components/hero/_macro.spec.js

```diff
@@ -105,15 +105,15 @@ describe('macro: hero', () => {
         expect(content).toBe('Topic Text');
     });
 
-    it('outputs the official statistics badge when officialStatisticsBadge is set to true', () => {
-        const $ = cheerio.load(renderComponent('hero', { ...EXAMPLE_HERO, officialStatisticsBadge: true }));
+    it('outputs the official statistics badge when officialStatisticsBadge is set to true and variants is `grey`', () => {
```

Comment:
Need the opposite of this to check that it doesnt render for the other variants, same with the census logo